document/installation.md

```diff
@@ -50,18 +65,25 @@ apt install -y g++ curl cmake pkg-config libcurl4-openssl-dev build-essential zl
   export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib
   ```
 
-- SRT library (If necessary)
+- SRT library version compatibility
+
+  ```bash
+  dpkg -l | grep libsrt
+  ii  libsrt-doc                 1.4.4-4                                 all          Secure Reliable Transport UDP streaming library (documentation)
+  ii  libsrt-openssl-dev:amd64   1.4.4-4                                 amd64        Secure Reliable Transport UDP streaming library (OpenSSL flavour development)
+  ii  libsrt1.4-openssl:amd64    1.4.4-4                                 amd64        Secure Reliable Transport UDP streaming library (OpenSSL flavour)
+  ```
 
-  The stable version can be installed with `apt install libsrt-dev` on `Ubuntu 20.04` version is `v1.4.0`.
-  **If SRT library is compiled from code, be sure that the code is reset to tag `v1.4.0`. Otherwise, srt_exporter library compiled would not work with installed SRT library on other devices. Make sure that SRT Exporter library is built and running based on the same version of SRT library.**  
+  > [!NOTE]
+  > **If SRT library is compiled from code, be sure that the code is aligned with tag `v1.4.X`. Otherwise, srt_exporter library would not get successfully compiled with installed SRT library.**
```

Comment:
having the [semantic version control](https://semver.org/) should be able to be compatible with any patch releases. that means, as long as that SRT version is `v1.4.X`, that should be compatible with other libraries.